project/src/components/pages/main-page/main-page.tsx

```diff
@@ -0,0 +1,169 @@
+                  {genre}
+                </span>
+                <span className="film-card__year">
+                  {year}
+                </span>
+              </p>
+              <div className="film-card__buttons">
+                <button className="btn btn--play film-card__button" type="button">
+                  <svg viewBox="0 0 19 19" width={19} height={19}>
+                    <use xlinkHref="#play-s" />
+                  </svg>
+                  <span>Play</span>
+                </button>
+                <button className="btn btn--list film-card__button" type="button">
+                  <svg viewBox="0 0 19 20" width={19} height={20}>
+                    <use xlinkHref="#add" />
+                  </svg>
+                  <span>My list</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <div className="page-content">
+        <section className="catalog">
+          <h2 className="catalog__title visually-hidden">Catalog</h2>
+          <ul className="catalog__genres-list">
+            <li className="catalog__genres-item catalog__genres-item--active">
+              <a href="#" className="catalog__genres-link">
+                All genres
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Comedies
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Crime
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Documentary
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Dramas
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Horror
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Kids &amp; Family
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Romance
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Sci-Fi
+              </a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">
+                Thrillers
+              </a>
+            </li>
+          </ul>
+          <div className="catalog__films-list">
+            {<MovieCard />}
```

Comment:
Тут и на несколько строк ниже скорее всего фигурные скобки лишние. Или так надо по макету?

```suggestion
            <MovieCard />
```